cpp/merkletree/serial_hasher.h

```diff
@@ -26,8 +27,8 @@ class SerialHasher {
   // Finalize the hash context and return the binary digest blob.
   virtual std::string Final() = 0;
 
-  // A virtual constructor.  The caller gets ownership of the returned object.
-  virtual SerialHasher* Create() const = 0;
+  // A virtual constructor.
```

Comment:
Yes, looks good.

Martin

On 17 May 2016 at 10:15, Pierre Phaneuf <notifications@github.com> wrote:

> In cpp/merkletree/serial_hasher.h
> <https://github.com/google/certificate-transparency/pull/1178#discussion_r63488970>
> :
>
> > @@ -26,8 +27,8 @@ class SerialHasher {
> >    // Finalize the hash context and return the binary digest blob.
> >    virtual std::string Final() = 0;
> >
> > -  // A virtual constructor.  The caller gets ownership of the returned object.
> > -  virtual SerialHasher* Create() const = 0;
> > +  // A virtual constructor.
>
> Expanded it a little, better?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/certificate-transparency/pull/1178/files/3906c7185afb535ae4a7f2e8944170ad3fdfbb50#r63488970>
>